src/modules/CRUD/services/CRUD.js

```diff
@@ -0,0 +1,50 @@
+import Api from '@terralego/core/modules/Api';
+
+export const fetchSettings = () => Api.request('crud/settings');
+
+export const getLayerFromCRUD = ({ menu = [] }, name) => {
```

Comment:
Is it really usefull to specify `fromCRUD` everywhere? We are in CRUD service scope, we should know that.